server/settings_test.go

```diff
@@ -618,20 +622,65 @@ groups:
 
 					res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
 						Body: server.ChangeSettingsBody{
-							AlertManagerURL:   url,
+							AlertManagerURL:   alertManagerURL,
 							AlertManagerRules: rules,
 						},
 						Context: pmmapitests.Context,
 					})
 					require.NoError(t, err)
-					assert.Equal(t, url, res.Payload.Settings.AlertManagerURL)
+					assert.Equal(t, alertManagerURL, res.Payload.Settings.AlertManagerURL)
 					assert.Equal(t, rules, res.Payload.Settings.AlertManagerRules)
 
 					gets, err := serverClient.Default.Server.GetSettings(nil)
 					require.NoError(t, err)
-					assert.Equal(t, url, gets.Payload.Settings.AlertManagerURL)
+					assert.Equal(t, alertManagerURL, gets.Payload.Settings.AlertManagerURL)
 					assert.Equal(t, rules, gets.Payload.Settings.AlertManagerRules)
 
+					t.Run("PrometheusReloaded", func(t *testing.T) {
+						prometheusBaseURL := pmmapitests.BaseURL.ResolveReference(&url.URL{
+							Path: "prometheus/",
+						})
+						client, err := prometheusApi.NewClient(prometheusApi.Config{
+							Address: prometheusBaseURL.String(),
+						})
+						require.NoError(t, err)
+
+						expected := prometheusApiV1.RuleGroup{
+							Rules: prometheusApiV1.Rules{
+								prometheusApiV1.AlertingRule{
+									Alerts:      []*prometheusApiV1.Alert{},
+									Annotations: model.LabelSet{"summary": "High request latency"},
+									Duration:    600,
+									Health:      "unknown",
```

Comment:
imho, this field should be ignored in the test because it's slightly related to what we are testing here. We are testing if prometheus reloaded configuration, not that prometheus is healthy or not. I'm suggesting to ignore it to make this test more stable and maybe faster(since we don't have to wait until prometheus is up). What do you think?

Reply:
Sounds good too